### ISSUE TITLE
Linguistic Analysis & Meeting Minutes compared to current Class Diagram

### ISSUE THREAD

**Issue by flyfl** [Beagle-PSE/Beagle#280]
(#248 #249 #250 #251 #252 #253 #254 #261 #271)
Sorted every difference/delta between (Linguistic Analysis + Meeting Minutes) and the current diagram state I've found under headwords.
("Optional" does not only address `optional`-marked features, but also features one could implicit read out from our SRS)

- Missing classes/attibutes
- Prototype classes/attributes
- Optional classes/attributes

## Missing classes/attributes
- [meeting drafts]`MeasurementOrder-class` specifying a single measurement or bundles of measurements
- [meeting drafts] Multiple Mapping from components (rdseffs, loops, branches) to `MeasurementResult` (-> `Blackboard`) 
- [meeting drafts] Multiple Mapping from components (rdseffs, loops, branches) to `StochasticExpression` (-> `Blackboard`) 
- [meeting drafts] Mapping from components (rdseffs, loops, branches) to the temporary best `StochasticExpression` (-> `Blackboard`) 
- [5. Functional Requirements /F50/]`AdaptiveTimeout` controlling unit that decides whether to stop a current measurment. Has a status enabled/disabled. 

## Prototype classes/attributes
- `GUI`Native Eclipse Features. Context sensitive menues. Optional Features: AnalysePoints set by user (/OD10/). User Timeout (/OD20/)
- `PCM Adapter` (Features: Save, annotate, read/stream PCM)

## Optional classes/attributes
(The following points are all subjective selected functions, that we may want to specify in the design or not)

- [implicit /A40/] `Comparator` between manually created PCM and afterwards automatically created PCM
- [implicit "4.1 Input"] `SourceCodeHandler`  that may include a logical connection to the Decorator model
- [/OD100/] Further resource demands (harddisk, network usage)
- [/OD130/] Measurement status data (to save current measurement)
- [/OD140/] Verification Data (checking whether input artefacts changed)
- [/OQ120/] Report Progress to the user
- [8.2 Challanges /C100/] `CPUHandler` that is connected to `MeasurementController` (Beagle aims to compensate CPU influencing performance factors)

**Comment by jGleitz** [comment]
The mappings are blocked by #263. Help is highly appreciated!

**Comment by jGleitz** [comment]
Please see if there is still anything missing. Take the things we discussed in meetings for some points into account.

**Comment by acBerger** [comment]
As this is probably out of date and we won't do it later "postponed" does not make sense. I'll close it.
